test-echo/README.md

```diff
@@ -15,17 +15,16 @@ how they can be used to gather and display a rich set of data
 ##### YAML
 
 First an aside about YAML - Each of the JSON metadata fields (`mccoy.schema_in`,`mccoy.schema_out`,`mccoy.info`)
-can be specified with JSON, as they have been in each precceding example, or with YAML (a superset of JSON that 
-allows us to drop a lot of the cumbersome symbols). This is useful for this example because our schemas become 
-quite lengthy, and the additional readability goes a long way. The one catch is because the way Dockerfile labels
-capture strings, one must explicitly insert newlines `\n`. See test/echo's [Dockerfile](Dockerfile) for examples. 
+can be specified with JSON, as they have been in each preceeding example, or with YAML (a superset of JSON that 
+is intuitively simpler).  Choosing YAML may make the most sense as schemas grow to greater complexity and readability matters.
+Please note that you must explicitly insert newlines `\n` to represent continuation for a label in the Dockerfile.

```

Comment:
this isn’t accurate, the backslash alone represents the continuation, and \n is needed so that yaml can interpret separate lines, therefore \n\ is needed at the end of every line